apps/backend/src/modules/users/user.controller.ts

```diff
@@ -24,6 +33,36 @@ class UserController extends BaseController {
 
 		this.userService = userService;
 
+		this.addRoute({
+			handler: (options) => {
+				return this.delete(
+					options as APIHandlerOptions<{
+						params: { id: string };
+					}>,
+				);
+			},
+			method: "DELETE",
+			path: UsersApiPath.$ID,
+			preHandler: combineHooks([
```

Comment:
```suggestion
			preHandler: [
```
you do not need to add an addition hook for it. you need to change the logic of the `preHandler` key to allow it accepts an array